crates/neon/src/sys/tsfn.rs

```diff
@@ -182,9 +182,12 @@ impl<T> Drop for ThreadsafeFunction<T> {
         }
 
         unsafe {
-            napi::release_threadsafe_function(
-                self.tsfn.0,
-                napi::ThreadsafeFunctionReleaseMode::Release,
+            assert_eq!(
```

Comment:
This is internal neon code and only prevents a memory leak. I think it would be better to ignore this error, possibly warn when a log, rather than panic.